sources/packages/backend/apps/api/src/route-controllers/supporting-user/supporting-user.supporting-users.controller.ts

```diff
@@ -204,8 +204,9 @@ export class SupportingUserSupportingUsersController extends BaseController {
     if (payload.offeringIntensity !== application.offeringIntensity) {
       throw new UnprocessableEntityException("Invalid offering intensity.");
     }
+    const submissionData = { ...payload, isAbleToReport: true };
```

Comment:
Is this the area supposed to be changed by @bidyashish in-progress work?

Reply:
It is suppose to be changed. I added this as minimal effort to ensure that this scope of PR does not hide the SIN input and read-only values displayed in the form when loaded from supporting users portal.